.spin/Procfile

```diff
@@ -0,0 +1,3 @@
+# This is an empty Procfile so that ui-extensions does not appear as a failed service in Spin.
+# See https://github.com/Shopify/spin/issues/3928
```

Comment:
Should we remove this link since it cannot be accessed publicly?

Reply:
I was mainly thinking that someone reading the code later could look at that issue and undo the workaround 🤷